autogenerated_examples/nxdl/python_scripts/h5py/ex_h5py_NXstxm.py

```diff
@@ -0,0 +1,186 @@
+ 
+root.create_group('untitled_entry')
+root['/untitled_entry'].attrs['NX_class'] = 'NXentry'
+root['/untitled_entry'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/'].create_group('untitled_instrument')
+root['/untitled_entry/untitled_instrument'].attrs['NX_class'] = 'NXinstrument'
+root['/untitled_entry/untitled_instrument'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('untitled_source')
+root['/untitled_entry/untitled_instrument/untitled_source'].attrs['NX_class'] = 'NXsource'
+root['/untitled_entry/untitled_instrument/untitled_source'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('monochromator')
+root['/untitled_entry/untitled_instrument/monochromator'].attrs['NX_class'] = 'NXmonochromator'
+root['/untitled_entry/untitled_instrument/monochromator'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('untitled_detector')
+root['/untitled_entry/untitled_instrument/untitled_detector'].attrs['NX_class'] = 'NXdetector'
+root['/untitled_entry/untitled_instrument/untitled_detector'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('sample_x')
+root['/untitled_entry/untitled_instrument/sample_x'].attrs['NX_class'] = 'NXdetector'
+root['/untitled_entry/untitled_instrument/sample_x'].attrs['EX_required'] = 'false'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('sample_y')
+root['/untitled_entry/untitled_instrument/sample_y'].attrs['NX_class'] = 'NXdetector'
+root['/untitled_entry/untitled_instrument/sample_y'].attrs['EX_required'] = 'false'
+ 
+root['/untitled_entry/untitled_instrument/'].create_group('sample_z')
+root['/untitled_entry/untitled_instrument/sample_z'].attrs['NX_class'] = 'NXdetector'
+root['/untitled_entry/untitled_instrument/sample_z'].attrs['EX_required'] = 'false'
+ 
+root['/untitled_entry/'].create_group('untitled_sample')
+root['/untitled_entry/untitled_sample'].attrs['NX_class'] = 'NXsample'
+root['/untitled_entry/untitled_sample'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/'].create_group('untitled_data')
+root['/untitled_entry/untitled_data'].attrs['NX_class'] = 'NXdata'
+root['/untitled_entry/untitled_data'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry/'].create_group('control')
+root['/untitled_entry/control'].attrs['NX_class'] = 'NXmonitor'
+root['/untitled_entry/control'].attrs['EX_required'] = 'false'
+ 
+root['/untitled_entry'].create_dataset(name='title', data='!some char data!', maxshape=None)
+root['/untitled_entry/title'].attrs['type'] = 'NX_CHAR'
+root['/untitled_entry/title'].attrs['units'] = ''
+root['/untitled_entry/title'].attrs['EX_required'] = 'true'
+ 
+root['/untitled_entry'].create_dataset(name='start_time', data='2021-03-22T14:00:28.221954', maxshape=None)
+root['/untitled_entry/start_time'].attrs['type'] = 'NX_DATE_TIME'
+root['/untitled_entry/start_time'].attrs['units'] = ''

```

Comment:
Does it make any sense for a time to have a unit?

Reply:
@prjemian says that such cases should have a 'units' attribute with the value 'NX_UNITLESS'

Reply:
units attribute removed